ROADMAP.md

```diff
@@ -1,15 +1,33 @@
 # Roadmap
 
-Here we draft the engineering roadmap of ChubaoFS
+Here we draft the engineering roadmap of ChubaoFS. 
 
+## Interfaces
 
-## Key Features
+FUSE-based POSIX inteface (done)
+
+S3-compatible object store (done)
+
+
+## Core Subsystems
+
+Authentication (done)
+
+Cross-zone replication (done)
+
+Reed-Solomon coding (April 2020)
```

Comment:
@bladehliu 

The development of Reed-Solomon coding cannot be completed in April 2020 from the current progress.